rotkehlchen/rotkehlchen.py

```diff
@@ -426,6 +426,10 @@ def unlock_user(
         Inquirer().inject_evm_managers([
             (ChainID.ETHEREUM, ethereum_manager),
             (ChainID.OPTIMISM, optimism_manager),
+            (ChainID.POLYGON_POS, polygon_pos_manager),
+            (ChainID.ARBITRUM_ONE, arbitrum_one_manager),
+            (ChainID.BASE, base_manager),
+            (ChainID.GNOSIS, gnosis_manager),
```

Comment:
What about scroll? Should it be all here? If yes then we need this to be an iteration of the `EVM_CHAINS_WITH_TRANSACTIONS`